src/ProductConstructionService/SubscriptionTriggerer/SubscriptionTriggererConfiguration.cs

```diff
@@ -0,0 +1,107 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using Maestro.Data;
+using Maestro.DataProviders;
+using Microsoft.ApplicationInsights.Channel;
+using Microsoft.ApplicationInsights.Extensibility;
+using Microsoft.DotNet.DarcLib.Helpers;
+using Microsoft.DotNet.DarcLib;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Logging;
+using Microsoft.DotNet.Kusto;
+using Azure.Storage.Queues;
+using Azure.Identity;
+using Microsoft.Extensions.Hosting;
+using Microsoft.Extensions.FileProviders;
+using Microsoft.DotNet.Internal.Logging;
+
+namespace SubscriptionTriggerer;
+public class SubscriptionTriggererConfiguration
+{
+    private const string ApplicationInsightsConnectionString = "APPLICATIONINSIGHTS_CONNECTION_STRING";
+    private const string AspnetcoreEnvironment = "ASPNETCORE_ENVIRONMENT";
+    private const string QueueConnectionString = "QueueConnectionString";
+    private const string ManagedIdentityClientId = "ManagedIdentityClientId";
+    private const string SqlConnectionStringUserIdPlaceholder = "USER_ID_PLACEHOLDER";
+    private const string DatabaseConnectionString = "BuildAssetRegistrySqlConnectionString";
+
+    public static IServiceCollection RegisterServices(IServiceCollection services, ITelemetryChannel telemetryChannel)
```

Comment:
I think the usual way is something like this:
```suggestion
    public static IServiceCollection RegisterSubscriptionTriggerer(this IServiceCollection services, ITelemetryChannel telemetryChannel)
```

And then in `Program.cs` instead of this  
`SubscriptionTriggererConfiguration.RegisterServices(services, telemetryChannel);`  
you do this:  
`services.RegisterSubscriptionTriggerer(telemetryChannel);`

Reply:
improved